lib/quick_travel/cache.rb

```diff
@@ -10,6 +10,8 @@ def self.cache(key, cache_options = {})
       cached_value = cache_store.read(key)
       return cached_value unless cached_value.nil?
       return nil unless block_given?
+      cache_options ||= {}
+      cache_options[:expires_in] = 1.day unless cache_options.key?(:expires_in)
```

Comment:
You could use fetch, it's the perfect example, as the presence of the key, even if set to nil, will force fetch to use the value.
```ruby
cache_options[:expires_in] = cache_options.fetch(:expires_in) { 1.day }
```